Dockerfile

```diff
@@ -1,26 +1,52 @@
-FROM --platform=$BUILDPLATFORM golang:1.19 as builder
+FROM docker.io/golang:1.19.7-alpine3.17 as builder
```

Comment:
The reason for using a full tag:
- When there is a new patch version, there is no change in the Dockerfile, and no rebuild is triggered, and nobody is aware that there is a new version with some bug fixes.
  Because of that, we have Dependabot, which creates PR on such new patch releases.
  This is how you can ensure that your container always includes the latest patches.
- When building software, binaries and containers, you aim to have a reproducible build, meaning you need to pin all dependencies. This way, it's easier to reproduce the behavior you can't pinpoint.
- When building containers, the OCI build tool may cache layers to decrease the building time. When only locked to the minor version, the build tool will use the existing cached version, despite a patch release available.